### DISCUSSION TITLE
Timeline notes

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#29] [Ideas / Wishlist]
Hi team,

Within each job, the timeline section could be used quite advantageously if we can get it to be automated each time someone makes any sort of change in each job ie: fleet change, date change etc.

This would allow for more streamlined and transparent communications within the system.

![StayOnHire 2](https://github.com/stayonhire/feedback/assets/141391069/eca3691a-b974-4b45-8d43-f87bb7f4f6ed)
![StayOnHire 1](https://github.com/stayonhire/feedback/assets/141391069/c8ac40b3-b8cf-4e7e-80cd-f9d9fa4856a5)

**Comment by dylanhru** [comment]
Hi @WadeRolfe,

Thanks for the idea! You mentioned 'fleet change' and 'date change'. To confirm I build the right thing:

**"Fleet change":** Is this in regards to performing an 'Offhire and swap out' function as pictured (in purple)?

**"Date change":** What type of date change situation? Like '_Asset X_' is allocated for 10 days, start date has moved ahead resulting in 9 days?


If I could do the above, what would the next **3 most valuable automated notifications** in the timeline be for you?

**Reply by WadeRolfe** [reply]
Hi Dylan,

Those are pretty good ideas along the exact lines I was thinking.

It's more of an indication that if a change is made to a job whether it be hire extension, machine swap out or partial off hire, it can be noted in the system automatically who did it and when they did it so that anyone can read the timeline and know who made the last change.

I would suggest the main 3 from my perspective would be:

- Equipment change out or swap out
- Hire extension date change
- Onhire/Offhire change (when someone pushes the button to onhire or offhire the job)

Hope that makes sense?

Thanks.
